src/pages/MovieDetail/components/InfoPage.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import ClickList from './ClickList';
+import MoreButton from './MoreButton';
+import GraphCard from './GraphCard';
+import GraphView from './GraphView';
+
+const InfoPage = ({ movieInfo }) => {
+  const [showMore, setShowMore] = useState(false);
+  const [text, setText] = useState('');
+
+  useEffect(() => {
+    setText(movieInfo.description);
+  });
+
+  return (
+    <InfoPageContainer>
+      {text && (
+        <InfoDetailPage>
+          <ClickListContainer>
+            {Cilck_List_Data.map(info => {
+              return <ClickList key={info.id} listName={info.listName} />;
+            })}
+          </ClickListContainer>
+          <MovieSummary>
+            <SummaryText>
+              <p className="title">{movieInfo.description_title}</p>
+              <p className="content">
+                {showMore ? text : `${text.substring(0, 30)}`}
+              </p>
+            </SummaryText>
+            <MoreBtn onClick={() => setShowMore(!showMore)}>
+              <MoreButton movieInfo={movieInfo} showMore={showMore} />
+            </MoreBtn>
+          </MovieSummary>
+          <MovieGraphContainer>
+            <GraphCard listName="실관람 평점" nameBottom={movieInfo.rating} />
+            <GraphCard
+              listName="예매율"
+              nameBottom={movieInfo.reservation_rate}
+            />
+            <GraphCard
+              listName="누적관객수"
+              nameBottom={movieInfo.total_viewer
+                .toString()
+                .replace(/\B(?=(\d{3})+(?!\d))/g, ',')}
+            >
+              <GraphView data={movieInfo.daily_viewer} />
+            </GraphCard>
+          </MovieGraphContainer>
+        </InfoDetailPage>
+      )}
+    </InfoPageContainer>
+  );
+};
+
+const InfoPageContainer = styled.div`
+  width: 100%;
+  height: auto;
+`;
+
+const InfoDetailPage = styled.div`
+  width: 80%;
+  height: auto;
+  margin: 0 auto;
+`;
+
+const ClickListContainer = styled.ul`
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  align-items: center;
+  text-align: center;
+  width: 100%;
+  height: 6.25rem;
+`;
+
+const MovieSummary = styled.div``;
+
+const SummaryText = styled.div`
+  width: 100%;
+  height: 6.25rem;
+
+  .title {
+    font-size: ${props => props.theme.fontSizes.titleSize};
+  }
+  .content {
+    margin-top: 0.9rem;
+    margin-bottom: 0.9rem;
+    font-size: ${props => props.theme.fontSizes.small};
+    color: rgba(68, 68, 68);
+  }
+`;
+
+const MoreBtn = styled.div`
+  margin-top: 0.9rem;
+  margin-bottom: 0.45rem;
+  border-bottom: 1px solid black;
+  opacity: 0.7;
+  text-align: center;
+
+  &:hover {
+    font-weight: bold;
+  }
+`;
+
+const MovieGraphContainer = styled.div`
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  width: 100%;
+  height: 20rem;
+  border: 1px solid rgba(68, 68, 68, 0.7);
+`;
+
+const Cilck_List_Data = [
```

Comment:
`UPPER_PASCAL_CASE`로 작성해주세요!
```suggestion
const CLICK_LIST_DATA = [
```